procedures/release-duty/merge-duty/merge_duty.rst

```diff
@@ -356,8 +314,8 @@ using an action payload such as:
    force-dry-run: false
    push: true
    behavior: bump-esr
-   to-branch: esr88
```

Comment:
I think `88` was targetting the next ESR when we initially added these docs. We guessed it'd be 88 but the calendar has changed and that became 91. So I think it should be `91` here. Nevertheless, once we EOL78, we'd need to revisit again so maybe it's worth adjusting these. Or maybe we should just use <CURRENT_ESR> and <NEXT_ESR> instead.